content/post/40-js-ipfs-0.30.md

```diff
@@ -0,0 +1,63 @@
+---
+date: 2018-07-09
+url: 40-js-ipfs-0-30
+title: js-ipfs 0.30.0 released
+author: Alan Shaw
+---
+
+Feature parity with go-ipfs just got a whole lot closer!
+
```

Comment:
Needs the fancy release tweet that people can RT

Reply:
Either one of 

- https://twitter.com/_alanshaw/status/1014990745508425728
- https://twitter.com/daviddias/status/1016364733107228673